editions/tw5.com/tiddlers/widgets/EventCatcherWidget.tid

```diff
@@ -7,16 +7,18 @@ type: text/vnd.tiddlywiki
 ! Introduction
 
 <<.from-version "5.1.23">>
-The event catcher widget traps JavaScript events dispatched within its child content, and allows invoking a series of ActionWidgets in response to the events. 
 
-In order for the events to be trapped they must:
+//This is a technically advanced widget intended for use by those familiar with HTML, CSS and DOM event handling.//
+
+The event catcher widget traps DOM-initiated Javascript events dispatched within its child content, and allows invoking a series of ActionWidgets in response to those events. 
 
-* be of one of the types specified as a parameter to the event catcher widget.
-* arise within a DOM node matching the selector specified as a parameter to the widget.
+In order for the events to be trapped they must:
 
-Use of the event catcher widget is beneficial when using large numbers of other trigger widgets such as the ButtonWidget is causing performance problems. The workflow it enables is akin to what is referred to as "event delegation" in JavaScript parlance. 
+* be of one of the events specified as a parameter to the event catcher widget's `events` attribute.
+* arise within a DOM node specified by the widget's `selector` attribute.
+* support [[event bubbling|https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#Event_bubbling_and_capture]].
 
-//This is an advanced widget intended to be used by those familiar with HTML, CSS and JavaScript.//
+Use of the event catcher widget is beneficial when using large numbers of other triggering widgets such as the ButtonWidget that may cause performance problems. The event catcher widget alleviates these problems by using an approach very similar to "event delegation" in JavaScript parlance. 
 
```

Comment:
I do not think it is appropriate to state that it alleviates the performance problems. It may help in some situations, but that will vary a lot depending on the architecture and where the actual performance bottleneck is, for instance on render or on refresh. This is why the original documentation is not so specific in this area. This needs to be revised to reflect that.

Reply:
Yes. Agreed. My least liked para of all.

Reply:
after reviewing again, I see what led me in that direction. The first sentence claims, "that may cause performance problems". This is kind of "out of nowhere" and not supported or countered by the rest of the (original) para. If the remainder does not alleviate/mitigate that claim, it simply serves no purpose, methinks. If the first sentence is removed or otherwise reworded (to not make the claim) then whole thing can be worded either very differently, or dropped. Since I don't know what/where your original intent was headed, I'll reinstate and back off.